app/web/template/payment_confirmation.gohtml

```diff
@@ -0,0 +1,22 @@
+{{ template "page" . }}
+
+{{ define "main" }}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <div class="govuk-panel govuk-panel--confirmation">
+                <h1 class="govuk-panel__title">
+                    {{ tr .App "paymentReceivedHeader" }}
+                </h1>
+                <div class="govuk-panel__body">
+                    {{ tr .App "yourRefNumber" }}<br><strong>{{ .PaymentReference }}</strong>
+                </div>
+            </div>
+
+            <h2 class="govuk-heading-m">{{ tr .App "thankYouHeader" }}</h2>
+
+            {{ trHtml .App "paymentConfirmationContent" }}
+
+            <a class="govuk-button" href="/task-list">{{ tr .App "continueLink" }}</a>
```

Comment:
change the href to `{{ link .App "/task-list" }}` so it handles welsh

Reply:
This is handy 👍